test/end2end_test.go

```diff
@@ -490,6 +490,7 @@ type test struct {
 	unaryServerInt              grpc.UnaryServerInterceptor
 	streamServerInt             grpc.StreamServerInterceptor
 	serverInitialWindowSize     int32
+	serverStaticWindow          bool
```

Comment:
Can this variable be given a more boolean like name like `isServerWindowStatic` or `useStaticServerWindow`?